graylog2-web-interface/src/components/users/UserDetails/SettingsSection.tsx

```diff
@@ -67,12 +67,14 @@ const StartpageValue = ({ type, id }: { type: string | null | undefined, id: str
 const SettingsSection = ({
   user: {
     timezone,
+    serviceAccount,
     sessionTimeout,
     startpage,
   },
 }: Props) => (
   <SectionComponent title="Settings">
     <ReadOnlyFormGroup label="Sessions Timeout" value={_sessionTimeout(sessionTimeout)} />
+    <ReadOnlyFormGroup label="Service Account" value={serviceAccount} />
```

Comment:
Regular reader users can set the "Service Account" attribute on their profile and effectively lock themselves out of the web interface. We might want to introduce a separate permission string for this flag or re-use one that only admin users have. (e.g., `users:edit`)

If users don't have the permission to change the value, it should also not show up in the form.